packages/PHPUnit/src/Rector/SpecificMethod/AssertIssetToSpecificMethodRector.php

```diff
@@ -35,10 +35,10 @@ public function getDefinition(): RectorDefinition
         return new RectorDefinition('Turns isset comparisons to their method name alternatives in PHPUnit TestCase', [
             new CodeSample(
                 '$this->assertTrue(isset($anything->foo));',
-                '$this->assertFalse(isset($anything["foo"]), "message");'
+                '$this->assertObjectHasAttribute("foo", $anything);'
             ),
             new CodeSample(
-                '$this->assertObjectHasAttribute("foo", $anything);',
+                '$this->assertFalse(isset($anything["foo"]), "message");',
```

Comment:
Why this change? Rule should end up with objectHasAttribute in the end

Reply:
@TomasVotruba The first `CodeSample` had two before codes, while the second two after codes, when it should be one before and one after:

![image](https://user-images.githubusercontent.com/2445045/65384727-f5455180-dd25-11e9-95f9-c724dd568007.png)

Reply:
Ah, I see, I didn't notice it's 2 code samples when reading on mobile phone. All right then :+1: